### PR TITLE
Update locale.lua

### PR DESCRIPTION
**Pull request by Kyramyz**
update french translation

### CODE REVIEW

**Comment by Stanzilla** [comment]
Your indentation is a bit broken

**Comment by Kyramyz** [comment]
This time i think it's good, sorry for the multiple updates. I start with Github

File: SavedInstances/locale.lua [hunk 1]
```diff
@@ -151,18 +160,32 @@ if locale == "frFR" then
     La["Time Left"] = "Temps restant "
     La["Tooltip Scale"] = "Taille de l'infobulle"
     La["Track"] = "Suivre"
+  	La["Track Mythic Keystone best run"] = "Suivre le meilleur temps pour la clé de donjon mythique"
```
Comment (Stanzilla) [inline SavedInstances/locale.lua]: here

File: SavedInstances/locale.lua [hunk 2]
```diff
@@ -151,18 +160,32 @@ if locale == "frFR" then
...
     La["Trade Skill Cooldowns"] = "Échéances des métiers"
     La["Trade skills"] = "Échéances des métiers"
     La["Transmute"] = "Transmutation"
     La["Type"] = "Type"
     La["Use class color"] = "Colorer par classe"
+  	La["Warn about instance limit"] = "Alerte sur la limite d'instance"
```
Comment (Stanzilla) [inline SavedInstances/locale.lua]: here

**Comment by Stanzilla** [comment]
Almost! And no worries :)

**Comment by Kyramyz** [comment]
Thank you, unfortunately, on my browser everything was displayed as correct ><

**Comment by Stanzilla** [comment]
Thanks!

**Pull request merged**